packages/allow-deny/allow-deny.js

```diff
@@ -439,8 +426,7 @@ CollectionPrototype._callMutatorMethod = function _callMutatorMethod(name, args,
   }
 
   const mutatorMethodName = this._prefix + name;
-  return this._connection.apply(
-    mutatorMethodName, args, { returnStubValue: true }, callback);
+  return await this._connection.applyAsync(mutatorMethodName, args, { returnStubValue: true , throwStubExceptions: true });
```

Comment:
This will probably break running multiple mutations right after each other on the client, for example:

```
Docs.insert({_id: '1'});
Docs.insert({_id: '2'});
```

Reply:
For sync methods, it should be the [same](https://github.com/meteor/meteor/blob/e5e1ad2349272b8dead4b84e3f82bd99a9747609/packages/allow-deny/allow-deny.js#L617-L645). For the async ones, we can test more, but I did change some things regarding this flow, in order to fix the mongo tests, and I think we'll be fine.